go/samples/http/main.go

```diff
@@ -47,9 +58,47 @@ func middleware(next http.Handler) http.Handler {
 	})
 }
 
-func main() {
+func runForMysql() *gosql.DB {
+	connection := "root:password@/sqlcommenter_db"
+	db = mysqldb.ConnectMySQL(connection)
```

Comment:
Can you add a comment above that this method would exit the application it it could not connect to the database. 
Another idea is to update this method name to "MustConnectMySQLOrDie". In that way the method would be clear, that the application would close if it could not connect with the MySQL database.

Reply:
Same for PG method also.